pages/veterinariansPage.ts

```diff
@@ -0,0 +1,12 @@
+import { Page } from '@playwright/test';
+import { HelperBase } from './helperBase';
+
+export class VeterinariansPage extends HelperBase{
+    constructor(page : Page){
+        super(page)
+    }
+    
+    async editVeterinarian(name: string){
```

Comment:
This method is only clicks "Edit Vet" button, make the method name to reflect it